spec/libsass-closed-issues/issue_2307/error-dart-sass

```diff
@@ -1,6 +1,6 @@
 Error: expected "{".
   ,
-3 | 
+3 | {
   | ^
   '
```

Comment:
This error doesn't make sense anymore. Expected "{" and it's pointing to a like that has one.

Reply:
Oh, good catch. This is another `source_span` bug: https://github.com/dart-lang/source_span/pull/30

Reply:
I'm not sure this is an improvement. Pointing to the position after the "{" makes it seem like the parser is expecting to see "{{" but would that actually fix the error? Should the error be a missing "}" instead?